docs/ops.html

```diff
@@ -586,7 +582,7 @@
   socket.request.max.bytes=104857600
   socket.receive.buffer.bytes=1048576
   socket.send.buffer.bytes=1048576
-  queued.max.requests=16
+  queued.max.requests=500
```

Comment:
@amethystic : Thanks for the patch. Looks good. Could you also remove num.replica.fetchers? I think the default value of 1 should work for most of the common cases.